dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java

```diff
@@ -339,8 +339,11 @@ private DependencyGraphResult levelOrder(
             for (ArtifactResult artifactResult :
                 resolutionException.getResult().getArtifactResults()) {
               if (artifactResult.getArtifact() == null) {
-                artifactProblems.add(new UnresolvableArtifactProblem(parentNodes));
```

Comment:
Memo:

This fails:

![image](https://user-images.githubusercontent.com/28604/74057557-2f03b580-49b2-11ea-9526-b521b703b93c.png)